xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -500,9 +500,8 @@ bool CPVRChannelGroups::AddGroup(const std::string &strName)
     if (!group)
     {
       // create a new group
-      group = CPVRChannelGroupPtr(new CPVRChannelGroup());
-      group->SetRadio(m_bRadio);
-      group->SetGroupName(strName);
+      group.reset(new CPVRChannelGroup(m_bRadio, CPVRChannelGroup::INVALID_GROUP_ID, strName, GetGroupAll()));
```

Comment:
Consider replacing this with `std::make_shared`:

```c++
group = std::make_shared<CPVRChannelGroup>(m_bRadio, CPVRChannelGroup::INVALID_GROUP_ID, strName, GetGroupAll());
```

https://clang.llvm.org/extra/clang-tidy/checks/modernize-make-shared.html

Reply:
Had that, resulted in linker error foe `CPVRChannelGroup::INVALID_GROUP_ID` on some platforms.

If you tell me how to solve the linker error I will be happy to change back to `std::make_shared`.

Reply:
That's a weird one! I'm happy to follow up after merging.

Reply:
Error details: https://jenkins.kodi.tv/view/Automation/job/BuildMulti-PR/12067/